code/go/0chain.net/smartcontract/dbs/event/writemarker.go

```diff
@@ -22,18 +23,15 @@ type WriteMarker struct {
 
 	AllocationRoot         string `json:"allocation_root"`
 	PreviousAllocationRoot string `json:"previous_allocation_root"`
+	FileMetaRoot           string `json:"file_meta_root"`
 	Size                   int64  `json:"size"`
 	Timestamp              int64  `json:"timestamp"`
 	Signature              string `json:"signature"`
 	BlockNumber            int64  `json:"block_number" gorm:"index:idx_wblocknum,priority:1;index:idx_walloc_block,priority:2"` //used in alloc_written_size
 
-	// file info
-	LookupHash  string `json:"lookup_hash" gorm:"index:idx_wlookup,priority:1"`
-	Name        string `json:"name" gorm:"index:idx_wname,priority:1;idx_walloc_file,priority:1"`
-	ContentHash string `json:"content_hash" gorm:"index:idx_wcontent,priority:1"`
```

Comment:
I see that associated changes for removing these 3 fields are also made in gosdk. But Since we are removing these fields, what happens when a client using older version gosdk try to send a write marker? Wouldn't the write marker validation fail, because are changing the fields on which the signature is generated

Reply:
@boddumanohar We are in development phase. Where are the clients?
Your comment would be valid if we had gone live.